tests/NewCypress/cypress/e2e/regression/pagination.cy.ts

```diff
@@ -0,0 +1,127 @@
+import paginationPage from "../../pages/paginationPage";
+
+describe('Pagination Tests', () => {
+
+
+    beforeEach(() => {
+        cy.login()
+        cy.visit('/search?keywords=west')
+    });
+
+    it('Should display multiple pagination buttons', () => {
+        paginationPage.getTotalPaginationButtons().should('be.greaterThan', 1);
+    });
+
+
+    it('Should navigate to a specific page when a page number is clicked on a large result page', () => {
+
+        cy.visit('/search?keywords=tru')
+
+        paginationPage
+            .getResults()
+            .then(firstPageFirstResult => {
+                const firstPageFirstResultText = firstPageFirstResult
+                    .first()
+                    .text();
+
+                paginationPage
+                    .clickPageNumber(2)
+                    .checkCurrentURLIsCorrect('pagenumber=2')
+                    .getResults()
+                    .first()
+                    .should('not.have.text', firstPageFirstResultText);
+
+                paginationPage
+                    .clickPageNumber(73)
+                    .checkCurrentURLIsCorrect('pagenumber=73')
+                    .getResults()
+                    .first()
+                    .should('not.have.text', firstPageFirstResultText);
+
+                paginationPage
+                    .clickPageNumber(72)
+                    .checkCurrentURLIsCorrect('pagenumber=72')
+                    .getResults()
+                    .first()
+                    .should('not.have.text', firstPageFirstResultText);
+
+            });
+
+    });
+
+    it('Should navigate to the next page on next button click', () => {
+        paginationPage.getResults().then(firstPageFirstResult => {
+            const firstPageFirstResultText = firstPageFirstResult.first().text();
+
+            paginationPage
+                .clickNext();
+
+            paginationPage
+                .getResults()
+                .first()
+                .should('not.have.text', firstPageFirstResultText);
+        });
+    });
+
+    it('Checks that the previous page button is not present on the first page of results', () => {
+        paginationPage
+            .checkPreviousButtonNotPresent()
+    });
+
+    it('Checks that the previous page button is not present on the first page of results', () => {
```

Comment:
```suggestion
    it('Checks that the next page button is not present on the last page of results', () => {
```

Reply:
Now updated this to say Next instead of previous - My bad I missed this one